lib/utils/frameView.cpp

```diff
@@ -0,0 +1,27 @@
+#include "frameView.hpp"
```

Comment:
I keep forgetting to say this, but we really should follow our own standards and use `PascalCase` for these files/classes.